modules/nf-core/fusionreport/detect/tests/nextflow.config

```diff
@@ -0,0 +1,10 @@
+process {
+    withName: 'FUSIONREPORT_DOWNLOAD' {
+        ext.args         =  "--no-cosmic"
```

Comment:
These parameters should be passed using `params.modules_args`.
You can find more info in https://nf-co.re/docs/tutorials/tests_and_test_data/nf-test_writing_tests